pytentiostat/config_reader.py

```diff
@@ -81,29 +158,108 @@ def get_cv_params(config_data):
 
 
 def get_exp_type(config_data):
+    """
+    Returns the type of experiment from the config data.
+
+    Parameters
+    ----------
+    config_data : dict
+        The configuration data
+
+    Returns
+    -------
+    exp_type : str
+        The type of experiment to be run.
+    """
+    
     exp_type = config_data["general_parameters"]["experiment_type"]
 
     return exp_type
 
 
 def get_exp_time(config_data):
+    """
+    Returns how long the experiment is to be run for the chronoamperometry case.
```

Comment:
why isn't this info returned with the chronoamperometry info?  Wouldn't that be better maybe?

Reply:
I just wrote this for plotting module since the only data it needs for the CA experiment is the time (not the voltage). But maybe this one is separate enough that I should just read from the dictionary in plotter and get rid of this function. Check for it could already be made by get CA params.